ambry-cloud/src/test/java/com.github.ambry.cloud/azure/AzureCloudDestinationTest.java

```diff
@@ -95,6 +98,26 @@
   private long creationTime = System.currentTimeMillis();
   private long deletionTime = creationTime + 10000;
   private long expirationTime = Utils.Infinite_Time;
+  private AzureReplicationFeed.FeedType azureReplicationFeedType;
+
+  /**
+   * Parameterized constructor.
+   * @param azureReplicationFeedType type of replication feed used by {@link AzureCloudDestination}
+   */
+  public AzureCloudDestinationTest(AzureReplicationFeed.FeedType azureReplicationFeedType) {
```

Comment:
It seems overkill to run every test with both feed types when only one test depends on it.  Please remove and make the two below findEntriesSince methods public.